main.go

```diff
@@ -192,7 +193,7 @@ func init() {
 	pf.StringVar(&logDir, "log.dir", getEnvVar("LOG_DIR", ""), "Custom log file directory.")
 	f.IntVar(&logRotateFilesToKeep, "log.rotate-files-to-keep", defaultLogRotateFilesToKeep, "Number of files to keep when rotating log files")
 	f.DurationVar(&logRotateInterval, "log.rotate-interval", defaultLogRotateInterval, "Time between log rotations (0 disables log rotation)")
-
+	f.StringVar(&advertisedEndpoint, "cluster.advertised-endpoint", "", "An external endpoint for this cluster")
```

Comment:
Not sure about "this cluster" since it is the coordinator/single.
Also we should talk about the format here, e.g. by adding an example.

Reply:
Of course, but it's the cluster's address, right? You mentioned yourself the load balancer scenario.

Reply:
it is correct, I just think the comment should be more elaborate.